src/containers/pages/OrganizationViews/serviceHooks.ts

```diff
@@ -0,0 +1,69 @@
+import {
```

Comment:
Do we have tests for this module somewhere?

Reply:
@moshthepitt There are no unit tests for this, they are however tested in the component tests which are more like integration tests. Will create an issue to add a test file to unit test this

Reply:
@p-netm thanks, please create an issue and I this PR is ready for merging IMHO